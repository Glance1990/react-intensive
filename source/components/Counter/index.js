import React from "react";

// Instruments
import Styles from "./styles.m.css";

const Counter = ({ count }) => (
    <section className = { Styles.counter }>Posts: {count}</section>
);

export default Counter;
