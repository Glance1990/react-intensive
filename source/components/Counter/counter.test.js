import React from "react";
import Counter from './';
import dom from 'react-test-renderer';

const renderTree = dom.create(<Counter count = { 3 } />).toJSON();

test('Counter component should correspond to their snapshot', () => {
    expect(renderTree).toMatchSnapshot();
});