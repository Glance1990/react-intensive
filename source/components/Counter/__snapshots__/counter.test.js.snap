// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component should correspond to their snapshot 1`] = `
<section
  className="counter"
>
  Posts: 
  3
</section>
`;
